common/init.sls

```diff
@@ -40,16 +40,14 @@ servo:
     - shell: /bin/bash
     - home: {{ common.servo_home }}
 
-{% for hostname, ip in common.hosts.items() %}
-host-{{ hostname }}:
-  host.present:
-    - name: {{ hostname }}
-    - ip: {{ ip }}
-{% endfor %}
-
 {% for ssh_user in common.ssh_users %}
 sshkey-{{ ssh_user }}:
   ssh_auth.present:
     - user: root
     - source: salt://{{ tpldir }}/ssh/{{ ssh_user }}.pub
 {% endfor %}
+
+/etc/hosts:
+  file.managed:
+    - contents: "127.0.0.1           localhost\n::1                 localhost\n255.255.255.255     broadcasthost"
+    - 
```

Comment:
Please also add `user: root` and `mode: 644`. 

The group is variable and depends on the OS, so we need to use Jinja templating to set it up. It should be `root` on Ubuntu and `wheel` on OS X.